### ISSUE TITLE
RichTextLabel theme updates do not update BOLD text 

### ISSUE THREAD

**Issue by jamie-pate** [godotengine/godot#49089]
**Godot version:**

3.3-stable

**OS/device including version:**

GLES3

**Issue description:**

When a theme is applied to a parent control, the theme's styles do not get applied to rich text label's *bold* text (maybe others) when bb_code is enabled and the text has styles

**Steps to reproduce:**
Open provided reproduction, click 'theme2' but the bold text style does not update. Click 'fix it' and it fixes it

**Minimal reproduction project:**

[RichTextLabelThemeRepro.zip](https://github.com/godotengine/godot/files/6543645/RichTextLabelThemeRepro.zip)

**Comment by jamie-pate** [comment]
Ugly workaround:

```Python
func _fix_rich_text_labels(control: Node):
	# https://github.com/godotengine/godot/issues/49089
	# richtextlabel: some font styles won't update when you change the theme
	if control is RichTextLabel:
		var rtl := control as RichTextLabel
		rtl.bbcode_text = rtl.bbcode_text
	for c in control.get_children():
		_fix_rich_text_labels(c)
```

**Comment by akien-mga** [comment]
Fixed by #49174.
